.template-lintrc.js

```diff
@@ -2,4 +2,8 @@
 
 module.exports = {
   extends: 'recommended',
+  rules: {
+    'no-whitespace-for-layout': false,
+    'require-input-label': false,
```

Comment:
Turned off these two new rules because they had a couple violations that didn't make sense to fix:

1) A test rendered an input without a label
2) 2 of the docs templates threw "extra whitespace" errors but it seemed like a bug. I couldn't fix it.

Reply:
Only 3 violations exist between the two of them, so they could be reenabled pretty easily if someone wants to attempt a fix.